### DISCUSSION TITLE
How to send workflow email with an attachment

### DISCUSSION THREAD

**Discussion by kseibi** [YetiForceCompany/YetiForceCRM#14847] [How to]
Hi, I have Yetiforce 6.1, I have created multiple workflows to automatically send emails whenever a status is changed and it's working very well, but I am having an issue to send emails with attachment automatically, I am sure it can be done through the workflow but I am having difficulties implementing it, could someone please let me know how to put an attachment to an email through workflow?

thank you very much.

**Comment by mariuszkrzaczkowski** [comment]
Workflows tasks:  "Send email template" + Related Documents

![image](https://user-images.githubusercontent.com/10198654/122507779-1fae6680-d001-11eb-82a5-16d171d2ce74.png)
